### PR TITLE
minor tweaks to sunpy projects

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: _projects/2023/sunpy/sunkit_image.md [hunk 1]
```diff
@@ -25,16 +25,25 @@ collaborating_projects:
...
-2. Performance profiling of the current functions to find bottle necks and improve performance where possible.
-3. Removal of ASDA.
+1. Refactor specific functions to accept and return a `sunpy.map.Map` (or an `NDcube` instead, to be decided during the project) instead of a `numpy.array`.
+   This would apply to functions that take a 2D data array and return one, functions like the `timelag` would not factor in to this.
+2. Tackle some of the following issues:
+   1. https://github.com/sunpy/sunkit-image/issues/83
+   2. https://github.com/sunpy/sunkit-image/issues/103
+   3. https://github.com/sunpy/sunkit-image/issues/76
+   4. https://github.com/sunpy/sunkit-image/issues/69
+   5. https://github.com/sunpy/sunkit-image/issues/83
```
[the remark this answers is not in the document]
Comment (nabobalis) [reply _projects/2023/sunpy/sunkit_image.md]: Fixed.

**Comment by bsipocz** [comment]
@nabobalis - is there anything particular holding this up, or could it be merged already?

**Comment by nabobalis** [comment]
> @nabobalis - is there anything particular holding this up, or could it be merged already?

My hope was to get more sunpy people to review it first.

**Comment by nabobalis** [comment]
If you can merge it, I think its ok for now.

**Review by bsipocz — APPROVED** [review]

**Comment by bsipocz** [comment]
Thanks @nabobalis!

**Pull request merged**